proxy.py

```diff
@@ -28,8 +29,11 @@ async def netcat(port):
     # Verbose mode
     if os.environ["VERBOSE"] == "1":
         command.append("-v")
-    command += [f"{mode}-listen:{port},fork,reuseaddr,max-children={max_connections}",
-                f"{mode}-connect:{ip}:{port}"]
+    if mode.lower() == "udp" and upd_answers == "0":
```

Comment:
Uppercased mode is not supported.

```suggestion
    if mode == "udp" and udp_answers == "0":
```

Reply:
May i ask the question why it's not supported? I couldn't find any reason that uppercase would not be supported. And the soccat options in the man page are all documented in upper case (see https://linux.die.net/man/1/socat) so i thought it would be reasonable for one that knows that this is a soccat wrapper to use `MODE: UDP` instead of `MODE: udp`

Reply:
It gives less cases to test -- when we hopefully have tests here in the future.

Reply:
it shouldn't be less cases to test, if it's not supposed to work, we are still supposed to test that it doesn't work -- in the future